src/store/plugins/usersWebSocketPlugin.js

```diff
@@ -2,34 +2,43 @@ import { usersEndpoint } from "@/services/API/Endpoints.js";
 const RECONNECT_TIME = 2000;
```

Comment:
need to fetch user details in App.vue so that if their status / orgs was updated when they were away - the change reflects when they come back.